beacon-chain/sync/rpc.go

```diff
@@ -88,3 +96,35 @@ func (r *RegularSync) registerRPC(topic string, base interface{}, handle rpcHand
 		}
 	})
 }
+
+func (r *RegularSync) chunkedHandler(stream libp2pcore.Stream, msg interface{}) error {
```

Comment:
Please put these handlers in a more appropriate go file. This `rpc.go` is meant for registering rpc topic handlers